tgui/packages/tgui/interfaces/NtosDeptOrder.tsx

```diff
@@ -114,23 +114,23 @@ const CooldownDimmer = () => {
           <Icon color="bug" name="route" size={20} />
         </Stack.Item>
         <Stack.Item fontSize="18px" color="orange">
-          Ready for another order in {time_left}...
+          Готовы к новому заказу через {time_left}...
         </Stack.Item>
         <Stack.Item textAlign="center" color="orange">
           <Button
             width="300px"
             lineHeight={2}
             tooltip={
               (!!can_override &&
-                'This action requires Head of Staff access!') ||
-              'Crate already shipped! No cancelling now!'
+                'Это действие требует доступа руководителя!') ||
+              'Ящик уже отправлен! Теперь отмена невозможна!'
             }
             fontSize="14px"
             color="red"
             disabled={!can_override}
             onClick={() => act('override_order')}
           >
-            <Box fontSize="22px">Override</Box>
+            <Box fontSize="22px">Ускорить</Box>
```

Comment:
```suggestion
            <Box fontSize="22px">Отменить</Box>
```
Насколько я помню, это позволяет главе отменить бесплатный заказ

Reply:
я так и не понял что кнопка делает. Override по идее ускоряет, но ингейм нихуя не понятно

Reply:
Я же сказал, отменяет текущий заказ, если шаттл еще не вызвали